packages/common/src/components/logos/Marko.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from 'react';
+
+export default ({ width = 32, height = 32, className }) => (
```

Comment:
Shouldn't this one come from the [templates repo](https://github.com/codesandbox/codesandbox-templates)? 🤔

Reply:
Same as above